packaging/test_irods_client_rest_cpp.py

```diff
@@ -452,3 +453,19 @@ def test_zone_report(self):
 
             assert(js0 == js1)
 
+    def test_changing_passwords_is_supported__issue_99(self):
+        # Show that the user can execute commands without error.
+        self.user.assert_icommand(['ils', '-ld'], 'STDOUT', [self.user.session_collection])
+
+        # Show that changing the user's password will require them to reauthenticate.
+        token = irods_rest.authenticate('rods', 'rods', 'native')
+        new_password = 'rods'
+        irods_rest.admin(token, 'modify', 'user', self.user.username, 'password', new_password, None, None, None)
```

Comment:
so this is the admin changing alice's password?

Reply:
yes